src/sensor/pac193x/include/Pac193x.h

```diff
@@ -118,10 +106,8 @@ pac193xErrorCode_t pac193xRefreshData(pac193xSensorConfiguration_t sensor);
  *
  * @param sensor[in] configuration of the sensor to use
  * @param info[out]  struct that holds the information read from the sensor
- * @return           returns the error code (0 if everything passed)
  */
-pac193xErrorCode_t pac193xGetSensorInfo(pac193xSensorConfiguration_t sensor,
-                                        pac193xSensorId_t *info);
+void pac193xGetSensorInfo(pac193xSensorConfiguration_t sensor, pac193xSensorId_t *info);
```

Comment:
turn [out] params into return values

Reply:
I checked the functions again. They return the sensor info through the info pointer, which is passed to the function.
So AFAIK there is no return value needed, right?

Reply:
@glencoe

Reply:
The idea is for the function to return the pac193xSensorId_t by value as in
```c
//...
pac193xSensorId_t info;
get_info(&info);
//... do more stuff
return info;
}